system-addon/lib/SectionsManager.jsm

```diff
@@ -40,13 +40,10 @@ const BUILT_IN_SECTIONS = {
       },
       button: {id: options.disclaimer_buttontext || "section_disclaimer_topstories_buttontext"}
     },
+    privacyNoticeURL: "https://www.mozilla.org/en-US/privacy/firefox/#suggest-relevant-content",
     maxRows: 1,
-    availableContextMenuOptions: ["CheckBookmark", "SaveToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl"],
-    infoOption: {
-      header: {id: options.provider_header || "pocket_feedback_header"},
-      body: {id: options.provider_description || "pocket_feedback_body"},
```

Comment:
These strings are now unused I believe (so add to the commit comment). Although _body is already missing… ???

Reply:
I think I saw them used in the Preferences panel. I’ll double check.

Reply:
I think "pocket_feedback_body" was a bug there. Probably was meant to be "pocket_description"? We use that in preferences. I'll fix it in `SectionManager.jsx`. But the pocket_feedback_header can die.